model/src/main/proto/deprecation.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package model;
+
+import "version.proto";
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Top-level proto used to store deprecation messages
+message DeprecationResponseDatabase {
+    // Stores the response to a request for an optional update.
```

Comment:
```suggestion
    // Stores a response that indicates that a user should be notified that an optional update is available.
```

Reply:
Same comment on clarity.

Reply:
Again, this object will store the response once a user is shown a dialog indicating that an optional update is available. The user only has two options, either update the app or dismiss the update. If they update the app, no response is stored. If they, however, choose to dismiss the update, this response will be stored on the DeprecationDatabase.

Reply:
Again, let me work on rephrasing the comment to make it clearer.